servicex/rabbit_adaptor.py

```diff
@@ -28,6 +28,7 @@
 # -*- coding: utf-8 -*-
 # pylint: disable=C0111,C0103,R0205
 import random
+import logging
```

Comment:
Are these logging changes related to markupsafe?

Reply:
No, I'm not sure how it got in here since I created a new branch of develop.  I'll resubmit a new PR that's off a clean branch